backend/settings.py

```diff
@@ -193,6 +194,10 @@
         'canvas_oauth': {
             'handlers': ['console'],
             'level': 'WARN'
+        },
+        'watchman': {
+            'handlers': ['console'],
+            'level': 'DEBUG',
```

Comment:
This feels like it should maybe use `os.getenv('DJANGO_LOG_LEVEL', 'INFO')` like above rather than everything at DEBUG level? Does this cause a lot of log messages when accessing watchman?

Reply:
I see these debug logs when the `/watchman/` url is called

```
ccm_web    | [2025-02-13 21:48:46 +0000] [DEBUG] [decorators.py:26] Checking '_check_database' for 'default'
ccm_web    | [2025-02-13 21:48:46 +0000] [DEBUG] [decorators.py:26] Checking '_check_database' for 'default'
ccm_web    | [2025-02-13 21:48:46 +0000] [DEBUG] [decorators.py:26] Checking '_check_cache' for 'default'
ccm_web    | [2025-02-13 21:48:46 +0000] [DEBUG] [decorators.py:26] Checking '_check_cache' for 'default'
```

Reply:
Not sure why duplicate logs of same check, since these are debug might now be noise in prod

Reply:
Yeah we'd want it to be able to run at `INFO `or `WARNING `probably in prod. Just seemed fine to match the DJANGO level.

Reply:
I think after the adjusting the Log level to Django log variable I don't see the duplication log messages